cosmo_tester/test_suites/test_blueprints/nodecellar_test_autoheal.py

```diff
@@ -139,14 +130,20 @@ def post_autoheal_assertions(self, before_state, after_state):
 
     def modify_blueprint(self):
         with YamlPatcher(self.blueprint_yaml) as patch:
-            patch.merge_obj('workflows', self.AUTOHEAL_WORKFLOW_YAML)
             patch.merge_obj('groups', self.AUTOHEAL_GROUP_YAML)
-            # FIXME: this is a workaround for multiple collectors starting
-            # multiple auto-heal workflows
-            patch.set_value("node_types.nodecellar\\.nodes\\.MonitoredServer."
+            patch.merge_obj("node_types.nodecellar\\.nodes\\.MonitoredServer."
```

Comment:
The node cellar example already has many collectors configured.
it would be best if instead of adding the example collector, we would simply update properties.service (in the groups section) to be something like "cpu\\.avg.*" (or whatever will be a valid regex that will catch exactly one metric)